README.rst

```diff
@@ -28,6 +28,10 @@ Long story short, ``UtcDateTime`` does:
 - ensure timestamps in database always to be encoded in UTC, and
 - work as you'd expect.
 
+A SQLAlchemy helper function, ``utcnow()``, is provided as an alternative
```

Comment:
Is SQLAlchemy pronounced sequel-alchemy rather than es-que-el-alchemy?  I'm just curious (as English is not my native tongue).

Reply:
[The author seems to indeed pronounce it in that way.](https://www.youtube.com/watch?v=P141KRbxVKc&t=700s) 😄

Reply:
I tend to pronounce SQL and SQLAlchemy as sequel and sequel-alchemy because it flows without pause when spoken. If I try to pronounce it es-que-el or es-que-el-alchemy, each "-" becomes a pause that slows the flow of speech.

It is a fairly common practice in English to phonetically pronounce acronyms instead of spelling them out.